.changeset/fluffy-falcons-refuse.md

```diff
@@ -0,0 +1,7 @@
+---
+'@cypress-design/react-icon': minor
```

Comment:
@emilmilanov I believe that the icon updates were considered as `patch` updates, instead of `minor` typically. We should probably continue to follow that pattern. Everything else looks good though.

Reply:
In looking at older PRs that introduce new icons, I saw both `minor` and `patch` used.

From a [semantic versioning](https://semver.org/) definition, I think this type of change would align with `minor`:  "when you add functionality in a backward compatible manner"

But I think you could argue either way.

Reply:
<img width="871" alt="image" src="https://github.com/user-attachments/assets/c6eb2a97-da94-46db-9e94-7b3880da66d3">

I used `yarn changeset` to create this part of the code; it doesn't offer a `patch` option, only major or minor. I can override this to patch if we agree on it. When I added icons to the DS before, I always used `minor`.

Reply:
@ryanjwilke There is no limit to the numbers.

@emilmilanov I did go back and look a little bit harder and the CHANGELOG.md files and it does seem like most of the changes that add icons use `patch`.  So I guess update your changeset file to use `patch` as well for all three packages.

Reply:
Changeset is updated from `minor` to `patch`. I also renamed icons to object-airplane-angled...